src/main/java/me/botsko/prism/actions/ItemStackAction.java

```diff
@@ -163,39 +169,77 @@ public void setItem(ItemStack item, int quantity, Map<Enchantment, Integer> ench
             }
         }
         if (meta instanceof FireworkEffectMeta) {
-            final FireworkEffectMeta fireworkMeta = (FireworkEffectMeta) meta;
-            if (fireworkMeta.hasEffect()) {
-                final FireworkEffect effect = fireworkMeta.getEffect();
-                if (effect != null) {
-                    if (!effect.getColors().isEmpty()) {
-                        final int[] effectColors = new int[effect.getColors().size()];
-                        int i = 0;
-                        for (final Color effectColor : effect.getColors()) {
-                            effectColors[i] = effectColor.asRGB();
-                            i++;
-                        }
-                        actionData.effectColors = effectColors;
-                    }
+            applyFireWorksMetaToActionData(meta);
+        }
+        if (meta instanceof BannerMeta) {
+            List<Pattern> patterns = ((BannerMeta) meta).getPatterns();
+            Map<String, String> stringyPatterns = new HashMap<>();
+            patterns.forEach(
+                    pattern -> stringyPatterns.put(pattern.getPattern().getIdentifier(), pattern.getColor().name()));
```

Comment:
Codacy found an issue: ['lambda arguments' has incorrect indentation level 20, expected level should be one of the following: 16, 18.](https://app.codacy.com/gh/AddstarMC/Prism-Bukkit/pullRequest?prid=6279951)